python/ngen_conf/src/ngen/config/init_config/value_unit_pair.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
 import re
-from typing import TYPE_CHECKING, Any, Generic, List, Optional, Type, TypeVar, Union
+from typing import TYPE_CHECKING, Any, Generic, List, TypeVar
```

Comment:
Earlier comment about imports is being at least partially resolved in this commit (review by commit as suggested, so didn't see this at the time.). However, I still see `List` where `list` was used in the previous commit, is `List` still needed?  Should probably double check consistent useage of `list` and `List`

Reply:
Yeah this is certainly a little confusing. When I was making these changes, I was under the impression that `pydantic` model's still needed to be annotated with types from the `typing` package. I was wrong about this... Only `Union` and `Optional` need to stay as _runtime_ type hints in pydantic modules.